### PR TITLE
Compile LESS using Grunt: add missing links, minor refactoring

### PR DESCRIPTION
**Pull request by dmytro-ch**
## Scope of this PR

- Editorial

## Purpose of this PR (required)

This PR adds missing links for "Compile LESS using Grunt" topic:
- [inherits]
- [LiveReload extension]
- [node.js]

## Affected URLs (required)

- https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/css-topics/css_debug.html
- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/css-topics/css_debug.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/frontend-dev-guide/css-topics/css_debug.md [hunk 1]
```diff
@@ -110,5 +110,6 @@ To change or override any of these variables, simply create a file in `<theme-di
 @font-family__sans-serif: 'Helvetica Neue', Helvetica, Arial, sans-serif;
 ```
 
-[customize the root source files or move the files included to the root files]: {{site.baseurl}}/guides/v2.2/frontend-dev-guide/css-topics/css-preprocess.html#css_exception
-[customize the root source files or move the files included to the root files]: {{site.baseurl}}/guides/v2.2/frontend-dev-guide/css-topics/css-preprocess.html#css_exception
+[inherits]: {{page.baseurl}}/frend-dev-guide/themes/theme-inherit.html
```
Comment (erikmarr) [inline guides/v2.2/frontend-dev-guide/css-topics/css_debug.md]:
```suggestion
[inherits]: {{page.baseurl}}/frontend-dev-guide/themes/theme-inherit.html
```

File: guides/v2.2/frontend-dev-guide/css-topics/css_debug.md [hunk 2]
```diff
@@ -110,5 +110,6 @@ To change or override any of these variables, simply create a file in `<theme-di
...
+[LiveReload extension]: http://livereload.com/extensions/
+[node.js]: https://github.com/joyent/node/wiki/installing-node.js-via-package-manager
```
Comment (erikmarr) [inline guides/v2.2/frontend-dev-guide/css-topics/css_debug.md]: Looks like this link has been archived. Should we link here instead?

https://github.com/nodejs/node

**Comment by dmytro-ch** [comment]
Hello @erikmarr,
thank you for reviewing this pull request.
I've applied the changes according to the recommendations.

Thank you!

**Comment by dobooth** [comment]
The links to node.js wiki redirect to https://github.com/nodejs/node 
Is that the correct destination?

**Comment by dmytro-ch** [comment]
Unfortunately, the original resource is achieved:
https://github.com/joyent/node/wiki/installing-node.js-via-package-manager

Now, this link is redirecting to the node.js repository https://github.com/nodejs/node, where user can find general information about node.js and installation process. So It should be the correct destination.

However, I think we still can use a link for the page that provides the same content as the old wiki resource.
https://nodejs.org/en/download/package-manager/

Thank you!

**Pull request merged**